app/models/pageflow/positioned_file.rb

```diff
@@ -34,7 +34,7 @@ def initialize
       end
 
       def thumbnail_url(*args)
-        ImageFile.new.attachment.url(*args)
+        ImageFile.new.thumbnail_url(*args)
```

Comment:
This broke a spec. Positioned file depends on getting the placeholder if the thumbnail is missing not the empty string. Maybe let's try to keep new changes as minimal as possible at this point.

Reply:
reverted in 61e91ea06b1c901a535a06ef9a66c78b7752eb20